qt/scientific_interfaces/ISISReflectometry/GUI/RunsTable/JobsViewUpdater.cpp

```diff
@@ -35,7 +35,18 @@ MantidWidgets::Batch::Cell qRangeCellOrDefault(RangeInQ const &qRangeInput, Rang
   return result;
 }
 
+boost::optional<size_t> incrementIndex(const Row &row) {
```

Comment:
Non-member functions cannot be `const`, as the function `const` qualifier dictates it will not change any member variables, but doesn't have any members it could even change

Reply:
Ah yes good point, think I got confused as I saw it in the header file

Reply:
No worries, you made me notice it didn't need to be in the header anyway